src/gui/MainWindow.h

```diff
@@ -103,6 +108,8 @@ class MainWindow : public GladeGui, public LayerCtrlListener
 	static void toggleMenuBar(MainWindow* win);
 
 	void createToolbarAndMenu();
+	
+	void initFloatingToolbox();
```

Comment:
This isn't called anywhere.

Reply:
oops... leftover from early stages.
Good catch. Thanks for looking over the code!